docs/sources/datasources/tempo/configure-tempo-data-source.md

```diff
@@ -247,7 +247,7 @@ datasources:
         queries:
           - name: 'Sample query'
             query: 'sum(rate(traces_spanmetrics_latency_bucket{$$__tags}[5m]))'
-      traceToProfiles:
+      tracesToProfiles:
```

Comment:
@grafana/observability-traces-and-profiling Is this update correct?

Reply:
It's correct